ClassAttendance/src/BLL/Automapper/AutomapperBLLConfig.cs

```diff
@@ -4,9 +4,9 @@
 
 namespace BLL.Automapper
 {
-    public class AutomapperConfig : Profile
+    public class AutomapperBLLConfig : Profile
```

Comment:
I guess it's useless because it's already differs by namespace.

Reply:
I decided to rename class instead of adding namespace to code
I think  services.AddAutoMapper(typeof(AutomapperConsoleConfig)) looks better than  services.AddAutoMapper(typeof(Bll.AutomapperConsoleConfig)). But actually not a big deal,

Reply:
Agreed.